### PR TITLE
Avoid hdiutils resource busy error in macos 13 builds

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 2b5226f4a6236431ab94b188730a3f94b9f10484: [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-5058-ec9de8e374a0e9bd651159d6394f37d60f529933-android-arm64.apk)
<details>
<summary>Other architectures</summary>

- [arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-5058-ec9de8e374a0e9bd651159d6394f37d60f529933-android-arm-neon.apk)
- [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-5058-ec9de8e374a0e9bd651159d6394f37d60f529933-android-x64.apk)
- [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-5058-ec9de8e374a0e9bd651159d6394f37d60f529933-android-x86.apk)
- [**all access arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-5058-ec9de8e374a0e9bd651159d6394f37d60f529933-android-arm64.apk)
- [all access arm-neon-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-5058-ec9de8e374a0e9bd651159d6394f37d60f529933-android-arm-neon.apk)
- [all access x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-5058-ec9de8e374a0e9bd651159d6394f37d60f529933-android-x64.apk)
- [all access x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield_all_access-dev-5058-ec9de8e374a0e9bd651159d6394f37d60f529933-android-x86.apk)
</details>

**Comment by m-kuhn** [comment]
Restarted a couple of times to check if it's stable

**Review by nirvn — APPROVED** [review]

File: .github/workflows/macos.yml [hunk 1]
```diff
@@ -129,6 +129,7 @@ jobs:
 
       - name: 📦 Package
         run: |
+          echo "Killing XProtect..."; sudo pkill -9 XProtect >/dev/null || true; # see https://github.com/actions/runner-images/issues/7522
```
Comment (nirvn) [inline .github/workflows/macos.yml:132]: @m-kuhn , do we need this for iOS?

Comment (m-kuhn) [reply .github/workflows/macos.yml:132]: No, it's the DMG creation -- specifically on macos-13 runners.

**Pull request merged**